doc/Media2.xml

```diff
@@ -2167,6 +2167,251 @@
+                [[tr2:AudioClipconfiguration]</para>
+              <para role="text"> Audio Clip Configuration to modify.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>response</term>
+            <listitem>
+              <para role="text">This is an empty message.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+			  <para role="param">env:Sender - ter:InvalidArgVal - ter:NoAudioClip</para>
+              <para role="text">The audio clip token does not exist.</para>
+              <para role="param">env:Sender - ter:InvalidArgVal - ter:InvalidConfig</para>
+              <para role="text">The configuration parameters are not possible to set.</para>         
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>access class</term>
+            <listitem>
+              <para role="access">WRITE_SYSTEM</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+	    <para> Note: Format of the audio clip cannot be changed during SetAudioClip operation.</para>
+      </section> 
+	  <section xml:id="section_wvd_dzg_ryf">
+        <title>DeleteAudioClip</title>
+        <para>This operation deletes audio clip configuration and associated audio clip in the device.
+		The audio clip should be de-associated with the event trigger if it is already associated before this operation.
+		This operation fails if the device is playing the audio clip same time due to user operation or schedule or event trigger.</para>
+        <variablelist role="op">
+          <varlistentry>
+            <term>request</term>
+            <listitem>
+              <para role="param">Token - 
+                [tt:ReferenceToken]</para>
+              <para role="text"> Token of the Audio Clip</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>response</term>
+            <listitem>
+              <para role="text">This is an empty message.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>faults</term>
+            <listitem>
+              <para role="param">env:Sender - ter:InvalidArgVal - ter:NoAudioClip</para>
+              <para role="text">The audio clip token does not exist.</para>
+              <para role="param">env:Sender - ter:InvalidArgVal - ter:AudioClipDeleteError</para>
+              <para role="text">The audio clip cannot be deleted due to device current usage.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>access class</term>
+            <listitem>
+              <para role="access">WRITE_SYSTEM</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </section>
+	  <section xml:id="section_wvd_dzg_ryg">
+        <title>PlayAudioClip</title>
```

Comment:
Better to split Play with a boolean to Play and Stop operations?

Reply:
we want to minimize the no of APIs overall, that is the reason play and stop are combined to single API.